paws/values.yaml

```diff
@@ -147,14 +147,32 @@ jupyterhub:
           import hashlib
           import subprocess
           import os
+          import json
           from oauthenticator.mediawiki import MWOAuthenticator
           from tornado import gen
+
+          from tornado.httpclient import AsyncHTTPClient
           
+
           class Auth(MWOAuthenticator):
               enable_auth_state = True
               def normalize_username(self, username):
                   return username
           
+              async def refresh_user(self, user, handler=None):
+                  client = AsyncHTTPClient()
+                  try:
+                      response = await client.fetch(f"https://meta.wikimedia.org/w/api.php?action=query&format=json&meta=globaluserinfo&guiuser={user.name}")
```

Comment:
This should probably set a user-agent per https://meta.wikimedia.org/wiki/User-Agent_policy

nit: I'd recommend setting `formatversion=2` in the request, makes the response sometimes a bit nicer to work with

Reply:
Nice one with `formatversion=2`, @supertassu allows for a more readable check on the next line.